test_files/enum/enum.tsickle.ts

```diff
@@ -26,7 +26,8 @@ let /** @type {number} */ enumStrIndex: number = EnumTest1[ /** @type {string} *
 function enumTestFunction(val: EnumTest1) {}
 enumTestFunction(enumTestValue);
 
-let /** @type {number} */ enumTestLookup = EnumTest1["XYZ"];
+let /** @type {number} */ enumTestLookup = EnumTest1.XYZ;
+let /** @type {?} */ enumTestLookup2 = EnumTest1["xyz".toUpperCase()];
```

Comment:
This is trouble, right? could we somehow warn/lint about this kind of computation in a property name?